koku/providers/aws/aws_provider.py

```diff
@@ -179,3 +179,7 @@ def cost_usage_source_is_reachable(self, credential_name, storage_resource_name)
                      topics_string, storage_resource_name)
         else:
             LOG.info('SNS is not configured for S3 bucket %s', storage_resource_name)
+
+    def infrastructure_type_meat(self, provider_uuid, tenant):
```

Comment:
I the `_meat` bit intentional?  I don't understand what it's supposed to mean.

Reply:
ha, I forgot to come back and search/replace for a better name ;)